dbms/src/Operators/Operator.cpp

```diff
@@ -162,7 +162,7 @@ OperatorStatus SinkOp::prepare()
     profile_info.anchor();
     auto op_status = prepareImpl();
 #ifndef NDEBUG
-    assertOperatorStatus(op_status, {OperatorStatus::NEED_INPUT});
+    assertOperatorStatus(op_status, {OperatorStatus::NEED_INPUT, OperatorStatus::IO_OUT});
```

Comment:
```suggestion
    assertOperatorStatus(op_status, {OperatorStatus::NEED_INPUT});
```
https://github.com/pingcap/tiflash/blob/c853837af6ecc170a854a19cc0a2d42cb12410aa/dbms/src/Operators/OperatorHelper.cpp#L29-L30

Reply:
Done